go.mod

```diff
@@ -13,6 +13,7 @@ require (
 
 require (
 	github.com/creack/pty v1.1.17 // indirect
+	github.com/davecgh/go-spew v1.1.1 // indirect
```

Comment:
what's this?

Reply:
Automatically added as a result of imports